domains/adeebkhan.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/akpathan2799/akpathan2799.github.io",
+    "owner": {
+        "username": "akpathan2799",
+        "email": "akpathan2799@gmail.com"
+       
+    },
+    "record": {
+        "CNAME": "akpathan2799.github.io"
```

Comment:
site gives 404 page

Reply:
what should i do i am not able to understand it

Reply:
help

Reply:
website gives a 404 page, aka it doesnt exist